katello/katello-backup

```diff
@@ -87,6 +72,66 @@ optparse = OptionParser.new do |opts|
   @dir = ARGV[0].dup
 end
 
+@databases.delete 'pgsql' if @is_foreman_proxy_content
+
+def specified_features
+  @options[:features] || []
+end
+
+def feature_included?(feature)
+  specified_features.include?(feature) || specified_features.include?("all")
+end
+
+def configure_configs
+  base_configs = [
+                   "/etc/foreman-proxy",
+                   "/etc/httpd",
+                   "/etc/foreman-installer",
+                   "/etc/pki/katello",
+                   "/etc/pki/katello-certs-tools",
+                   "/etc/pki/pulp",
+                   "/etc/pulp",
+                   "/etc/puppet",
+                   "/etc/qpid",
+                   "/etc/qpid-dispatch",
+                   "/root/ssl-build",
+                   "/var/www/html/pub",
+                   "/etc/squid",
+                   "/etc/puppetlabs",
+                   '/opt/puppetlabs/puppet/cache/foreman_cache_data',
+                   '/opt/puppetlabs/puppet/ssl/',
+                   '/var/lib/puppet/foreman_cache_data',
+                   '/var/lib/puppet/ssl'
+                 ]
```

Comment:
The indentation here seems weird to me, bit of anitpick but this feels more natural:

```
  base_configs = [
     "/etc/foreman-proxy",
     "/etc/httpd",
     .........
     '/var/lib/puppet/foreman_cache_data',
     '/var/lib/puppet/ssl'
 ]
```